### DISCUSSION TITLE
Breaking? Are they?

### DISCUSSION THREAD

**Discussion by jeremytammik** [jeremytammik/RevitLookup#157] [General]
Hi Roman, in the release announcements, you mention 'breaking changes', e.g., in [release 2024.0.3](https://github.com/jeremytammik/RevitLookup/releases/tag/2024.0.3). Why do you say 'breaking'? The feature you list there looks new to me, and I don't think it breaks any existing functionality. Do you just mean interesting, or main, significant, or what would be the best term?

**Comment by Nice3point** [comment]
Important, major. Sorry for my English, I renamed it "General". I think that would be better. Just general descriptions of the main changes

**Comment by jeremytammik** [comment]
_General_ would mean general, as in generic. If you want to highlight the most important changes, I would call it _main_ or _significant_. Or simply _most important_. Here is how I (very slightly) reformulated your release notes for the blog: https://thebuildingcoder.typepad.com/blog/2023/04/nice-accelerator-and-compiling-the-revit-2024-sdk.html#2
